pyscisci/embedding.py

```diff
@@ -0,0 +1,437 @@
+# -*- coding: utf-8 -*-
+"""
+.. module:: embedding
+    :synopsis: The main Embedding Class
+
+.. moduleauthor:: Jisung Yoon <jisung.yoon92@gmail.com>
+ """
+import sys
+import numpy as np
+from scipy import sparse
+import networkx as nx
```

Comment:
move networkx to optional dependency, this is the only function to use it (and you really only convert out of nx graphs).